.ci/docker-compose.override.yml

```diff
@@ -0,0 +1,51 @@
+version: '3'
+
+services:
+
+  logstash:
+    command: /usr/share/plugins/plugin/.ci/logstash-run.sh
+#    build:
+#      args:
+#        - ELASTIC_STACK_VERSION=$ELASTIC_STACK_VERSION
+#        - DISTRIBUTION=${DISTRIBUTION:-default}
+#        - DISTRIBUTION_SUFFIX=${DISTRIBUTION_SUFFIX}
+#    environment:
+#      - DISTRIBUTION=${DISTRIBUTION:-default}
+#      - DISTRIBUTION_SUFFIX=${DISTRIBUTION_SUFFIX}
+#      - INTEGRATION=${INTEGRATION:-false}
+#      - SECURE_INTEGRATION=${SECURE_INTEGRATION:-false}
+
+  elasticsearch:
+#    image: docker.elastic.co/elasticsearch/elasticsearch:$ELASTIC_STACK_VERSION
```

Comment:
don't you want to use the env variable here?
noticed in the README guide that you do `export ELASTIC_STACK_VERSION=7.7.0`

Reply:
Your are right, was piece of config that I commented while tinkering with it

Reply:
Having the env var `ELASTIC_STACK_VERSION` read by config, when it has value like `7.x` the build fails. The question is, should the AppSearch and Elasticsearch version used aligned to this or it should be only Logstash that need to bring that version?

Reply:
for example in circleci config inside `app-search-php` they use a fixed version of the Elasticsearch and AppSearch (https://github.com/elastic/app-search-php/blob/master/.circleci/config.yml#L29) probably updating it on every release